exercises/custom-set/example.js

```diff
@@ -1,20 +1,38 @@
 export default class CustomSet {
   constructor(data = []) {
     this.data = {};
-    data.forEach(el => this.put(el));
-    this.comparator = typeof data[0] === 'number' ?
-      (a, b) => a - b
-      : (a, b) => b <= a
-      ? 1 : -1;
+    data.forEach(el => this.add(el));
+  }
+
+  add(el) {
+    this.data[el] = el;
+    return this;
   }
 
   delete(el) {
     delete this.data[el];
     return this;
   }
 
+  clear() {
+    this.data = {};
+    return this;
+  }
```

Comment:
`clear` function can be removed from the implementation since it is not being used anywhere now.